jupyter-book/trajectories/pseudotemporal.ipynb

```diff
@@ -15,53 +15,95 @@
    ]
```

Comment:
<p>"A pseudotime"</p><p>Reads a bit weird to me. But might technically not be wrong.</p><p> </p><p>"initial cell"</p><p>Could maybe also mention the term "root cell" here already? Think it's quite common.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<blockquote>Reads a bit weird to me. But might technically not be wrong.</blockquote><p>I updated it to <em>Pseudotimes rank [...]</em>.</p><blockquote>Could maybe also mention the term "root cell" here already? Think it's quite common.</blockquote><p>Done. Might be a bit nested now, though.</p>